src/smaccm-flight/src/SMACCMPilot/Flight/Control/Altitude.hs

```diff
@@ -168,20 +170,25 @@ monitorAltitudeControl attrs = do
                       0.0
                       dt
 
+                  store vz_fb thrust_cmd
+
                   -- thrust_cmd is unbounded, so make sure it is
                   -- within the limits [0.1-1]
-                  thrust_cmd_norm   <- call fconstrain (0.1) 0.9 thrust_cmd
+                  --thrust_cmd_norm   <- call fconstrain (0.1) 0.9 thrust_cmd
+                  --store vz_fb_norm thrust_cmd_norm
```

Comment:
If we're constraining later on in the code now, let's remove this commented code